package.json

```diff
@@ -136,14 +136,14 @@
   "workspaces": [
     "configs/eslint-config-mongosh",
     "configs/tsconfig-mongosh",
+    "packages/java-shell",
+    "packages/js-multiline-to-singleline",
```

Comment:
🤔 My bootstrap keeps doing this. I'll remove this change from this pr, maybe it's a version issue or I have some old workspaces leftover in my project?